src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.FrameworkReferenceResolution.targets

```diff
@@ -229,21 +233,78 @@ Copyright (c) .NET Foundation. All rights reserved.
       <ComHostSourcePath>@(ResolvedComHostPack->'%(Path)')</ComHostSourcePath>
     </PropertyGroup>
 
-  </Target>
-  
-  <UsingTask TaskName="ResolveRuntimePackAssets" AssemblyFile="$(MicrosoftNETBuildTasksAssembly)" />
-  
-  <Target Name="ResolveRuntimePackAssets" DependsOnTargets="ResolvePackageAssets"
-          Condition="'@(RuntimePack)' != '' And '$(DesignTimeBuild)' != 'true'">
-    
-      <GetPackageDirectory
-        Items="@(RuntimePack)"
-        PackageFolders="@(AssetsFilePackageFolder)">
+    <GetPackageDirectory
+      Items="@(RuntimePack)"
+      PackageFolders="@(AssetsFilePackageFolder)">
 
       <Output TaskParameter="Output" ItemName="ResolvedRuntimePack" />
+
+    </GetPackageDirectory>
+
+    <ResolveFrameworkReferences
+      FrameworkReferences="@(FrameworkReference)"
+      ResolvedTargetingPacks="@(ResolvedTargetingPack)"
+      ResolvedRuntimePacks="@(ResolvedRuntimePack)">
+
+      <Output TaskParameter="ResolvedFrameworkReferences" ItemName="ResolvedFrameworkReference" />
```

Comment:
What metadata does this item have?

Reply:
Ah, its here: https://github.com/dotnet/sdk/pull/3355/commits/fb563e3d76f0fb8d1321cf201c74ccf9e831e87e#diff-6cfbacaa410e7ee1787b4771b158ded0R42.

Yes, this appears to be what we want. We'll also want a ResolveFrameworkReferences in MSBuild proper so that we can call regardless of we're an SDK project or not.

Reply:
Can you show me an example of one of these and we can decide what properties we want to show up?